app/views/galleries/index.rb

```diff
@@ -91,7 +98,19 @@ def published_gallery_topic_options
     end
 
     def galleries_content
-      @galleries.map { |gallery| gallery_content(gallery) }
+      res = @galleries.map { |gallery| gallery_content(gallery) }
```

Comment:
Could you restore `galleries_content` to what it was before, then move this clicktip-adorning code into a separate method, that you might call like `add_info_clicktip(galleries_content))`?